src/readabilityScore/Main.java

```diff
@@ -0,0 +1,42 @@
+package readabilityScore;
+
+public class Main {
+
+    private static void initializeCalculator(IndexCalculator calculator, Text text) {
+        calculator.setText(text);
+        calculator.calculateIndex();
+        calculator.determineAge();
+        calculator.displayInformation();
+    }
+    private static void processCommand(String command, Text text) {
+        if ("all".equals(command)) {
+            System.out.println();
+            double ageSumm = 0;
+            for (Index index : Index.values()) {
+                final var calculator = index.getCalculator();
+                initializeCalculator(calculator, text);
+                ageSumm += calculator.determineAge();
+            }
+            double averageAge = ageSumm / Index.values().length;
+            System.out.printf("\nThis text should be understood in average by %.2f-year-olds.\n", averageAge);
+        } else {
+            final var index = Index.valueOf(command);
```

Comment:
Can we process a case when the user enters a wrong command that cannot be converted to the enum?